package.json

```diff
@@ -71,7 +72,8 @@
     "js-base64": "^3.7.7",
     "md5": "^2.3.0",
     "pino": "^8.19.0",
-    "semver": "^7.5.4"
+    "semver": "^7.5.4",
+    "uuid": "^8.3.2"
```

Comment:
this is a fairly old version of uuid (4 years old) but the last one that's compatible with our module settings

Reply:
if we don't want to add an additional dependency, we can delegate the event ID generation to consuming libraries (js-client, node, etc)

Reply:
Are you hoping to use uuid v7 because of this https://buildkite.com/resources/blog/goodbye-integers-hello-uuids/ or another feature?

Reply:
unfortunately we can't use uuid v7 because it's not available in the version of the library we're using. looks like it was added on [uuid](https://github.com/uuidjs/uuid) v10. however yes, that could be a pretty decent primary key for these events. I don't want to make any assumptions here, the only goal was to have a unique identifier and we can't use sequential IDs reliably because we can't guarantee the sequence ordering (we'd have to persist additional state for the last ID used, which is inconvenient).